From 8ef447d3eeb4e3d68dffac42da2b9101829d14c2 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:43 -0400
Subject: [redhat] kabi: Adding symbol proc_mkdir (fs/proc/generic.c)

Message-id: <2d74b8923b0756717996cef60548c98b92dfc723.1410510773.git.poros@redhat.com>
Patchwork-id: 96108
O-Subject: [PATCH RHEL7.1 163/233] kabi: Adding symbol proc_mkdir (fs/proc/generic.c)
Bugzilla: 1062361 1071099
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1062361,1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    This function is used to create proc directory.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/proc_mkdir
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/proc_mkdir
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/proc_mkdir

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/proc_mkdir b/redhat/kabi/kabi-module/kabi_ppc64/proc_mkdir
new file mode 100644
index 0000000..60a7223
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/proc_mkdir
@@ -0,0 +1,2 @@
+#1-
+0x73e390ad proc_mkdir vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/proc_mkdir b/redhat/kabi/kabi-module/kabi_s390x/proc_mkdir
new file mode 100644
index 0000000..9504ff4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/proc_mkdir
@@ -0,0 +1,2 @@
+#1-
+0xf969cfe0 proc_mkdir vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/proc_mkdir b/redhat/kabi/kabi-module/kabi_x86_64/proc_mkdir
new file mode 100644
index 0000000..d56da7a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/proc_mkdir
@@ -0,0 +1,2 @@
+#1-
+0x73839c7a proc_mkdir vmlinux EXPORT_SYMBOL
-- 
1.7.1